packages/nocodb-sdk/src/lib/formulaHelpers.spec.ts

```
@@ -74,4 +74,67 @@ describe('Formula parsing and type validation', () => {
 
     expect(result1.dataType).toEqual(FormulaDataTypes.NUMERIC);
   });
+
+  describe.only('Date and time interaction', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)